storage/src/test/java/org/apache/kafka/tiered/storage/integration/OffloadAndConsumeFromLeaderTest.java

```diff
@@ -127,10 +127,10 @@ protected void writeTestSpecifications(TieredStorageTestBuilder builder) {
                  *       - For topic B, only one segment is present in the tiered storage, as asserted by the
                  *         previous sub-test-case.
                  */
-                .bounce(broker)
+                // .bounce(broker)
```

Comment:
Bouncing the broker updates the log-start-offset to first-local-log-segment-base-offset in [LogLoader](https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/log/LogLoader.scala#L183), so the remote log segments are removed due to breach by log-start-offset. Will fix this issue as part of KAFKA-15351.